hvac/v1/__init__.py

```diff
@@ -946,11 +946,11 @@ def renew_token(self, token=None, increment=None, wrap_ttl=None):
         """
         params = {
             'increment': increment,
+            'token': token,
```

Comment:
Perhaps it makes more sense to move setting this `token` key in the params dict to under the following conditional? I.e., no need to send it in the params if we're doing the renew-self call:

```python
def renew_token(self, token=None, increment=None, wrap_ttl=None):
    params = {
        'increment': increment,
    }

    if token is not None:
        params['token'] = token
        return self._adapter.post('/v1/auth/token/renew', json=params, wrap_ttl=wrap_ttl).json()
    else:
        return self._adapter.post('/v1/auth/token/renew-self', json=params, wrap_ttl=wrap_ttl).json()

```

Reply:
Yes sure! :+1: